### PR TITLE
Add moment-recur@1.0.5 w/ git auto-update

### PR DESCRIPTION
**Pull request by extend1994**
Pull request for issue: #10325

Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Profile of the lib
 * Git repository (required): https://github.com/c-trimm/moment-recur
 * Official website (optional, not the repository): 
 * NPM package url (optional): 
 * License and its reference: [Unlicense](https://github.com/c-trimm/moment-recur#license)
 * GitHub / Bitbucket popularity (required):
   - Count of watchers: 9
   - Count of stars: 140
   - Count of forks: 62
 * NPM download stats (optional):
   - Downloads in the last day:
   - Downloads in the last week:
   - Downloads in the last month:

# Essential checklist
 * [ ] I'm the author of this library
   * [ ] I would like to add link to the page of this library on CDNJS on website / readme
 * [x] This lib was not found on cdnjs repo
 * [x] No already exist / duplicated issue and PR
 * [x] The lib has notable popularity
   * [x] More than 100 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [ ] More than 500 downloads stats per month on npm registry
 * [x] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [x] Has valid tags for each versions (for git auto-update)
 * [x] Auto-update setup
 * [x] Auto-update target/source is valid.
 * [x] Auto-update filemap is correct.

# Git commit checklist
 * [x] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [x] The parent of the commit(s) in the PR is not old than 3 days.
 * [x] Pull request is sending from a non-master branch with meaningful name.
 * [x] Separate unrelated changes into different commits.
 * [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [x] Close corresponding issue in commit message
 * [x] Mention related issue(s), people in commit message, comment.

### CODE REVIEW

File: ajax/libs/moment-recur/1.0.5/moment-recur.js [hunk 1]
```diff
@@ -0,0 +1,744 @@
...
+      throw Error("Can't find moment");
+    }
+
+    // Interval object for creating and matching interval-based rules
+    var Interval = (function() {
+        function createInterval(units, measure) {
+            // Make sure all of the units are integers greater than 0.
+            for (var unit in units) {
+                if (units.hasOwnProperty(unit)) {
+                    if (parseInt(unit, 10) <= 0) {
```
Comment (PeterDaveHello) [inline ajax/libs/moment-recur/1.0.5/moment-recur.js]: https://github.com/c-trimm/moment-recur/blob/v1.0.5/moment-recur.js#L24
```js
                     if ( parseInt(unit, 10) <= 0 ) {
```


Obviously different indent here ... @cdnjs/intern3 you should use md5 and sha to make sure the main files are 100% the same as upstream.

File: ajax/libs/moment-recur/package.json [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+  "author": "Casey Trimm",
+  "license": "Unlicense",
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/c-trimm/moment-recur.git",
+    "fileMap": [
+      {
+        "basePath": "",
+        "files": [
+          "moment-recur*.+(css|map|js)"
```
Comment (sufuf3) [inline ajax/libs/moment-recur/package.json]: Is this really good to use this pattern?  What do you say?

Comment (PeterDaveHello) [inline ajax/libs/moment-recur/package.json]: I think you should tell the reason why not, the origin question is meaningless.

**Comment by extend1994** [comment]
@PeterDaveHello I updated `moment-recur.js`, please help me review it, thank you.
And the difference is due to the last `moment-recur.js` is from `master` branch of the library, not v1.0.5. I will be careful afterward.

**Pull request merged**